Sources/Extensions/Foundation/URLExtensions.swift

```diff
@@ -55,5 +56,34 @@ public extension URL {
 	public mutating func appendQueryParameters(_ parameters: [String: String]) {
 		self = appendingQueryParameters(parameters)
 	}
+}
 
+#if os(iOS) || os(tvOS)
+import UIKit
+// MARK: - Methods
+public extension URL {
+        
+    /// SwifterSwift: Generate a thumbnail image from given url. Returns nil if no thumbnail could be created. This function may take some time to complete. It's recommended to dispatch the call if the thumbnail is not generated from a local resource.
+    ///
+    ///        var url = URL(string: "https://video.golem.de/files/1/1/20637/wrkw0718-sd.mp4")!
+    ///        let image_at_0seconds = url.thumbnail()
+    ///        let image_at_5seconds = url.thumbnail(fromTime: 5)
+    ///
+    ///        DisptachQueue.main.async {
+    ///            someImageView.image = url.thumbnial()
+    ///        }
+    ///
+    /// - Parameter fromTime: Seconds into the video where the image should be generated.
+    public func thumbnail(fromTime time: Float64 = 0) -> UIImage? {
+        let asset: AVAsset = AVAsset(url: self)
+        let imageGenerator = AVAssetImageGenerator(asset: asset)
+        let time = CMTimeMakeWithSeconds(time, 1)
+        
+        var actualTime: CMTime = CMTimeMake(0, 0)
```

Comment:
Same with `actualTime`

Reply:
Sure 👍 However that slipped in the code.